### PR TITLE
Issue #108 - Fix cascade soft delete with newly installed Extension

### LINKED ISSUE #108: Add SoftDeletableListenerExtensionBundle
Nainstalovat podla https://github.com/E-vence/SoftDeleteableListenerExtensionBundle#install
Pre kazdy foreign entitu pridat `@Evence\onSoftDelete(type="CASCADE")` alebo ` @Evence\onSoftDelete(type="SET NULL")`

### PR DESCRIPTION
**Pull request by nitrajka**
I also fixd unappropriate db column names and variables in entities
Resolves #108

### CODE REVIEW

File: src/src/AdminBundle/Entity/Registration.php [hunk 1]
```diff
@@ -184,39 +188,39 @@ public function setAttendee($attendee)
     /**
      * Get the value of eventDetails.
      */
-    public function getEvents()
+    public function getEvent()
```
Comment (ZoltanOnody) [inline src/src/AdminBundle/Entity/Registration.php:191]: je super, ze si to premenovala ale toto rozbilo polko applikacie.

Comment (nitrajka) [reply src/src/AdminBundle/Entity/Registration.php:191]: Nerozbilo :) dam na to pozor. Radsej mi povedz, co mam spravit s tym konfliktom.
Chce to po mne aby som si tipla, ktory hash ta ma ostat.

Comment (ZoltanOnody) [reply src/src/AdminBundle/Entity/Registration.php:191]: hmmm... rebasni na mastra, vymaz ten file a potom ho vygeneruj nanovo

Comment (nitrajka) [reply src/src/AdminBundle/Entity/Registration.php:191]: ako sa generuje na novo?

**Comment by ZoltanOnody** [comment]
Attendee.php

```
 /**
     * @ORM\ManyToOne(targetEntity="Language", inversedBy="attendees")
     * @ORM\JoinColumn(name="language_id", referencedColumnName="id")
     */
    public $languages;
```

tu si to asi zabudla pridat

**Comment by ZoltanOnody** [comment]
UnsubscribeController riadok 24 `$language = $registration->getLanguages();`

**Comment by ZoltanOnody** [comment]
- [x] UnsubscribeController riadok 24 `$language = $registration->getLanguages();`
- [x] ConfirmRegistrationController riadok 23
- [x] Default controller riadok 69: $registration->setLanguages($attendeeLanguage);
- [x] AddMockedDataCommand riadok 232

**Comment by ZoltanOnody** [comment]
Default controller riadok 69: $registration->setLanguages($attendeeLanguage);

**Comment by ZoltanOnody** [comment]
AddMockedDataCommand riadok 232

**Comment by ZoltanOnody** [comment]
v Registration prepis `setEvents` na `setEvent` a potom si oprav chyby v 

- [x] DefaultController riadok 105
- [x] AddMockedDataCommand riadok 222
- [x] ConfirmRegistraionController riadok 24
- [x] UnsubsribeController riadok 25
- [x] UnsubsribeController riadok 26

**Review by ZoltanOnody — APPROVED** [review]

File: src/src/AdminBundle/Entity/Registration.php [hunk 2]
```diff
@@ -52,22 +53,25 @@ class Registration extends BaseEntity
...
-     * @ORM\JoinColumn(name="attendee_id", referencedColumnName="id")
+     * @ORM\JoinColumn(name="attendee_id", referencedColumnName="id", nullable=false)
      */
     private $attendee;
 
     /**
+     * @Evence\onSoftDelete(type="CASCADE")
      * @ORM\ManyToOne(targetEntity="Event", inversedBy="registrations")
-     * @ORM\JoinColumn(name="event_details_id", referencedColumnName="id")
+     * @ORM\JoinColumn(name="event_id", referencedColumnName="id", nullable=false)
```
Comment (ZoltanOnody) [inline src/src/AdminBundle/Entity/Registration.php:65]: @nitrajka toto rozbilo uplne vsetko!!!!!!!!!!!!!!!!!

**Pull request merged**